Changelog.md

```diff
@@ -1,5 +1,9 @@
 ### 0.6.8 (unreleased)
 
+Important Bugfixes:
+ * Code Generator: Fixed that string literals could sometimes cause invalid IR code to be generated due to missing escaping when the ABIEncoderV2 was active.
```

Comment:
Hm, I would say 'invalid' is the wrong word - it implies that it will be a compilation error. "incorrect" might be better here.

Reply:
Also are you sure this only applies to ABIEncoderV2?

Reply:
Well, strictly speaking there was an compilation error, just not on the sol code but the generated yul code

Reply:
non-the-less updated to use "incorrect" now